go.mod

```diff
@@ -304,5 +304,6 @@ replace (
 	// fix potential security issue(CVE-2020-26160) introduced by indirect dependency.
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
+	github.com/pingcap/kvproto => github.com/CabinfeverB/kvproto v0.0.0-20230721052823-f43cf27c6650
```

Comment:
Should update `github.com/pingcap/kvproto` and do not use replace before merge.

Reply:
I know. Now is to be able to review at the same time